components/sidebar.js

```diff
@@ -0,0 +1,91 @@
+import Button from '../components/button';
+
+export default function Sidebar(props) {
+
+  function profilePhoto() {
+    if (props.user.photoUrl) {
+      return props.user.photoUrl;
+    } else {
+      return ('https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_640.png');    }
+  }
+
+  return (
+    <div className="sidebar">
+        <div className="profile">
+          <div className="profile-photo-container">
+              <img className="profile-photo" src={profilePhoto()} alt="profile photo" />
+          </div>
+          <h2 className="username">{props.user.name.firstName + " " + props.user.name.lastName}</h2>
+          <h3 className="role">{props.user.role}</h3>
+          <Button
+              size="small"
+              text="update profile"
+              color="#ffdacc"
+              backgroundColor="#00C1B8"
+              onClick={props.openUpdateModal}/>
+          <div className="profile-info-div">
+            <p><b>Goals: </b>{props.user.goals}</p>
+            <p><b>Experience: </b>{props.user.experience}</p>
+            <p><b>{(props.user.role === "mentor")?"Expertise: ": "Skills: "} </b>{props.user.skills}</p>
+            {(props.user.role === "mentor") ? "" : <p><b>Portfolio: </b>{props.user.portfolioUrl}</p>}
```

Comment:
Non blocking: It could make this code more readable by grouping the mentor specific fields in a `mentorFields` function that returns JSX and conditionally rendering like so:
```js
{props.user.role && mentorFields()}
```

That would be in place of each ternary.

Reply:
I was able to group the fields:
https://github.com/heyellieday/mentor-weekly/commit/eec83a2727a9ae089ef5cd048f6e61cf8a53bcda#diff-bc9285499c412a6333c2da2dac466313